app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import app.errors
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if "vaccine" not in visitor:
+            raise app.errors.NotVaccinatedError("Visitor is not vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] <= datetime.date.today():
```

Comment:
Change this condition, the `expiration_date` should be less than `datetime.date.today()`, not less than equal.